js/sitnlayers.js

```diff
@@ -32,26 +33,40 @@
         {
           layer: '1-plan_ville',
           matrixSet: _crs
-        });
-        source = new ol.source.WMTS(sitnLayers.WMTSOptions)
-        sitnLayers.sitnCurrentBaseLayer.setSource(source)
-    });
+        })
+      let source = new ol.source.WMTS(sitnLayers.WMTSOptions)
+      sitnLayers.sitnCurrentBaseLayer.setSource(source)
+    })
 
     sitnLayers.view = new ol.View({
       projection: _projection,
       center: ol.proj.fromLonLat([6.80, 47.05], _projection),
       zoom: 4
-    });
+    })
 
     sitnLayers.map = new ol.Map({
       target: 'sitn-map',
```

Comment:
Shouldn't the target be configurable ?

I would rather call `sitnLayers('target-map')` to show them how to load it dynamically into a given `<div>`...